components/CharityList.js

```diff
@@ -1,29 +1,55 @@
-import React, { Component, useState, useEffect } from 'react'
```

Comment:
Added the database fetch to CharityList.js since this is the component that renders the data